filebeat/module/elasticsearch/audit/test/test-audit.log

```diff
@@ -5,3 +5,5 @@
 {"@timestamp":"2018-10-31T09:35:12,303", "node.id":"DSiWcTyeThWtUXLB9J0BMw", "event.type":"transport", "event.action":"access_granted", "user.name":"elastic", "user.realm":"reserved", "user.roles":["superuser"], "origin.type":"rest","origin.address":"[::1]:61711", "action":"cluster:admin/xpack/security/user/change_password", "request.name":"ChangePasswordRequest"}
 {"@timestamp":"2018-10-31T09:35:12,314", "node.id":"DSiWcTyeThWtUXLB9J0BMw", "event.type":"transport", "event.action":"access_granted", "user.name":"_xpack_security", "user.realm":"__attach", "user.roles":["superuser"], "origin.type":"local_node", "origin.address":"127.0.0.1:9300", "action":"indices:admin/create", "request.name":"CreateIndexRequest", "indices":[".security-6"]}
 {"@timestamp":"2019-01-27T20:15:10,380", "node.name":"node-0", "node.id":"y8fa3M5zSSGo1M_KJRMUXw", "event.type":"rest", "event.action":"authentication_success", "user.name":"elastic-admin", "origin.type":"rest", "origin.address":"[::1]:58955", "realm":"default_file", "url.path":"/_search", "request.method":"GET", "request.body":"\n{\n    \"query\" : {\n        \"term\" : { \"user\" : \"kimchy\" }\n    }\n}\n", "request.id":"WzL_kb6VSvOhAq0twPvHOQ"}
+{"timestamp": "2021-02-10T14:24:38,466Z", "type": "server", "level": "INFO", "component": "o.e.x.i.a.TransportPutLifecycleAction", "cluster.name": "docker-cluster", "node.name": "d739441e55ef", "message": "adding index lifecycle policy [kibana-event-log-policy]", "cluster.uuid": "qb5P5M_QSbWRsE1xA9MDKA", "node.id": "lz9sNb0TRCy9PO99_5aOkA"  }
+{"type":"server","timestamp":"2021-02-10T14:34:34,598Z","level":"INFO","component":"o.e.x.i.IndexLifecycleTransition","cluster.name":"docker-cluster","node.name":"d739441e55ef","message":"moving index [ilm-history-3-000001] from [{\"phase\":\"hot\",\"action\":\"unfollow\",\"name\":\"wait-for-yellow-step\"}] to [{\"phase\":\"hot\",\"action\":\"rollover\",\"name\":\"check-rollover-ready\"}] in policy [ilm-history-ilm-policy]","cluster.uuid":"qb5P5M_QSbWRsE1xA9MDKA","node.id":"lz9sNb0TRCy9PO99_5aOkA"}
```

Comment:
@sayden Are you sure these are audit log lines? They look like ES server log lines — notice the `"type": "server"` in them. How did you generate them?